### PR TITLE
fix: add nair0lf32

### LINKED ISSUE #1207: EDEMESSI Florian

### PR DESCRIPTION
**Pull request by nair0lf32**
resolve #1207

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/nair0lf32.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/35136136/175789287-a0796936-1bf8-443c-8adc-f56c4ebb95bc.png",
+    "issueId": 1207,
+    "name": "EDEMESSI Florian",
```
Comment (schmelto) [inline profiles/nair0lf32.json:4]:
```suggestion
    "name": "Edemessi Florian",
```

File: profiles/nair0lf32.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "username": "nair0lf32"
```
Comment (Panquesito7) [inline profiles/nair0lf32.json:5]: Looks like the indentation here is wrong.
```suggestion
  "image": "https://user-images.githubusercontent.com/35136136/175789287-a0796936-1bf8-443c-8adc-f56c4ebb95bc.png",
  "issueId": 1207,
  "name": "EDEMESSI Florian",
  "username": "nair0lf32"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge :tada: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
Awesome, thanks! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Elegant

**Pull request merged**